runtime/lua/vim/shared.lua

```diff
@@ -6,6 +6,10 @@
 
 local vim = vim or {}
 
+function vim.empty_dict()
```

Comment:
This does not make any sense. `vim._empty_dict_mt` exists only in in process lua. You would need to extend libmpack-lua with the corresponding functionality.